config/data_sync_list/common.json

```diff
@@ -5,6 +5,20 @@
             "Description": "Host State Persisted data",
             "SyncDirection": "Active2Passive",
             "SyncType": "Immediate"
+        },
```

Comment:
Commit message:
> Add config data for fan presence and system logging

Please use `config: ` tag in the commit message header.

I feel fan presence and logging could be added as separate commits.

> as noted in internal documentation

Please avoid statement like this in the open source projects.

Please test details in the commit message. (I am not sure whether data is exist but, I assume you verified that configuration works as expected.)